src/content/nl/2020/security.md

```diff
@@ -892,7 +889,8 @@ Er kunnen veel verschillende factoren zijn die van invloed zijn op de beveiligin
 {{ figure_markup(
   image="security-adoption-of-https-per-country.png",
   caption="Overname van HTTPS per land",
-  description="Staafdiagram met het percentage sites waarop HTTPS is ingeschakeld, voor sites die aan verschillende landen zijn gerelateerd Zwitserland, Nieuw-Zeeland, Ierland, Nigeria en Australië staan met 95% tot 94% in de top vijf. Aan de andere kant staan Thailand, Iran, Zuid-Korea, Taiwan en Japan op 76% tot 72%.",
+  description="Staafdiagram met het percentage sites waarop HTTPS is ingeschakeld, voor sites die aan verschillende landen zijn gerelateerd.
+Zwitserland, Nieuw-Zeeland, Ierland, Nigeria en Australië staan met 95% tot 94% in de top vijf. Aan de andere kant staan Thailand, Iran, Zuid-Korea, Taiwan en Japan op 76% tot 72%.",
```

Comment:
Don’t see why this should be on separate lines. Any chance you could correct here and in English and Japanese?